stylesheets/typography.css

```diff
@@ -6,3 +6,55 @@
     font-size: 62.5%;
 }
 
+/* headings */
+.heading{
+    font-weight: 700;
+}
+
+.h1{
+    font-size: 3rem;
+}
+
+.h2{
+    font-size: 2.5rem;
+}
+
+.h3{
+    font-size: 2rem;
+}
+
+.h4{
+    font-size: 1.75rem;
+}
+
+.h5{
+    font-size: 1.5rem;
+}
+
+.h6{
+    font-size:1.25rem;
+}
+
+/* para */
+.para{
+    font-weight: 400;
+}
```

Comment:
You can give this a more generic name like font-md

Reply:
thanks for pointing out , fixed in recent commit